src/reason_lexer.mll

```diff
@@ -446,6 +446,8 @@ rule token = parse
         lexbuf.lex_curr_p <- { curpos with pos_cnum = curpos.pos_cnum - 1 };
         STAR
       }
+  | "#" (appropriate_operator_suffix_chars | "#")
```

Comment:
This will only work with operators that are two characters long, right? I think you need to use a "?" after the parens.